### ISSUE TITLE
Fonts not working - Asian Characters and Boxes

### ISSUE THREAD

**Issue by ConStambo** [fontello/fontello#631]
This is happening on Chrome.

I imported the fonts and I'm only seeing Asian characters and illegal character boxes.  After searching, there were some support tickets on the Enfold theme website relating to that. Enfold theme is for WordPress and this is the theme that I am using. However, this does not seem to be the theme and it seems to do with the downloads from the Fontello website. For instance, select a few icons and then click on customise codes. In the middle of each box you will see the icon, however above you will see either Asian characters or illegal character boxes.

Additionally, Asian characters are also showing up in google search results.

Hope this gets fixed soon!

**Comment by ConStambo** [comment]
On Microsoft Edge. It's stuck on loading... but in Google search results it's showing as English.

**Comment by puzrin** [comment]
Please provide mode detailed info to prove that problem is not related to outside things.

1. Make sure problem is reproducible  with demo page, included into fontello archive (+ how to reproduce)
2. Attach downloaded archive or config file.

**Comment by ConStambo** [comment]
Loaded eventually on Mozilla and MS Edge but took a few minutes. Also, I tested the demo and it was fine and so I figured I must have been doing something wrong then. My bad! Looks like I wasnt adding "font family" in the css. 

My apologies for that but I was looking for a solution everywhere on the internet and didn't come across one, just people asking about the same issue so I thought it may be a bug. Especially after seeing the same characters in the chrome browser window on the website.

Thank you for your time!
